### ISSUE TITLE
Create MultiCamera calibration samples and tutorials

### ISSUE THREAD

**Issue by SatjaSivcev** [zivid/python-samples#70]

**Comment by ebruun** [comment]
Are there plans for Python-specific multi-camera calibration and stitching samples/tutorials?

**Comment by SatjaSivcev** [comment]
Yes, there are plans for multi-camera calibration and stitching implemented in python.
We have this task in our backlog but we haven't decided yet when we will implement it.
I suppose you have seen our C++ implementation https://github.com/zivid/zivid-cpp-samples/tree/master/source/Applications/Advanced/MultiCamera?

**Comment by srsidd** [comment]
Any update on getting a python based tutorial for point cloud stitching? (I have seen the C++ implementation one, but it'll be nice to have a python sample)

**Comment by SatjaSivcev** [comment]
This is now done. The Python samples include both multi-camera calibration and stitching:

- `source/applications/advanced/multi_camera/` — `multi_camera_calibration.py`, `multi_camera_calibration_from_zdf.py`, `stitch_by_transformation.py`, `stitch_by_transformation_from_zdf.py`
- `source/applications/advanced/stitching/` — `stitch_continuously_rotating_object.py`, `stitch_using_robot_mounted_camera.py`, `stitch_via_local_point_cloud_registration.py`

Closing.
